src/components/GameBoard/Piece/Piece.tsx

```diff
@@ -65,20 +61,16 @@ const Piece: FC<PieceProps> = ({
             isActive={isActive}
           ></PencilGrid>
         )}
-        {number === 0 ? "" : isPaused ? "" : `${number}`}
+        {number === 0 ? "" : `${number}`}
       </div>
     </>
   )
 }
 
-const mapStateToProps = (state: InitialState) => ({
-  isPaused: selectGameIsPaused(state),
-})
-
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   onPieceClick: index => dispatch(createBoardSelectPieceAction(index)),
   setHighlightedNumber: number =>
     dispatch(createBoardSetHighlightedNumber(number)),
 })
 
-export default connect(mapStateToProps, mapDispatchToProps)(Piece)
+export default connect(mapDispatchToProps)(Piece)
```

Comment:
```suggestion
export default connect(null, mapDispatchToProps)(Piece)
```

Reply:
You still need the first parameter, since the second parameter is for the `mapDispatchToProps`.